target/linux/generic/hack-5.10/661-use_fq_codel_by_default.patch

```diff
@@ -31,12 +31,11 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  struct Qdisc_class_common {
 --- a/net/sched/Kconfig
 +++ b/net/sched/Kconfig
-@@ -4,8 +4,9 @@
- #
+@@ -5,7 +5,9 @@
  
  menuconfig NET_SCHED
--	bool "QoS and/or fair queueing"
-+	def_bool y
+ 	bool "QoS and/or fair queueing"
```

Comment:
With this change NET_SCHED is not an anonymous config option any more. 
I would prefer if you would remove the title test like before:
```
bool
default y
```

Reply:
The warning should be triggered because bool doesn't have a description. Am I wrong?

Reply:
It is anyway activated in our generic config, so it should not make a difference:
```
target/linux/generic/config-5.10:CONFIG_NET_SCHED=y
target/linux/generic/config-5.4:CONFIG_NET_SCHED=y
```

Reply:
IIUC, this means we can simply drop "def_bool y" and restore those lines as per upstream, only keeping from the original patch:
```
+ select NET_SCH_FQ_CODEL
```

Could we resolve this then? That false warning leads people (like me) to look for some problem to fix.